app/build.gradle

```diff
@@ -62,6 +62,15 @@ def getHVRMLApiKey = { ->
     return ""
 }
 
+def getHVRMLSpeechServices = { ->
+    if (getHVRMLApiKey().isEmpty()) {
+        return "{ com.igalia.wolvic.speech.SpeechServices.MEETKAI }"
+    } else {
+        return "{ com.igalia.wolvic.speech.SpeechServices.HUAWEI_ASR, " +
+                "com.igalia.wolvic.speech.SpeechServices.MEETKAI }"
+    }
+}
+
```

Comment:
I think this approach is more flexible in case we want to add or remove services for particular configurations, or change the default service.

It also makes it easier to write code that iterates through the available speech recognition services, which can be found at `BuildConfig.SPEECH_SERVICES`.